### DISCUSSION TITLE
Russian geolock with Wink

### DISCUSSION THREAD

**Discussion by DigitalDancer** [iptv-org/community#611] [Q&A] [answered]
I have tried out quite a few Russian regional network links that used to be region free last year.  Now they are region locked, through a company or service called Wink (my actual region is USA) Which keeps insisting the stream or site is only for Russia even with Russian VPN (I have tried Proton and Vyper for this purpose ... works fine on Russian state tv channels like Russia One) and even used a GPS spoofer. I have had mixed results with GPS spoofing so it could be that.  Also, TV-IN.RU  main stream is fine except during KHL matches ... (this is the regional channel for HC Metallurg,  in Magnitogorsk) Any ideas?

**Comment by BellezaEmporium** [comment]
Possibly that @Dum4G or @freearhey can answer this one. As far as I know, WINK is a sort of copyright enforcer, making sure geo-blocking works fine.

**Reply by DigitalDancer** [reply]
I also tested some of the Russian only streams in the Russian playlist on this site ... most of them seemed fine with a VPN that supports Russia, but one of them (can't read Russian, so don't know what is about) was WINK locked, even though it wasn't even 720P ... might be worth checking out for anyone who wants to test a way around this issue.

**Reply by freearhey** [reply]
I'm not very familiar with Wink. As I understand from their [website](https://wink.ru/), it is a regular service that, among other things, provides broadcasts of TV channels. But maybe it also participates in geo-blocking of Russian channels in general, I don't know.

**Reply by BellezaEmporium** [reply]
> I'm not very familiar with Wink. As I understand from their [website](https://wink.ru/), it is a regular service that, among other things, provides broadcasts of TV channels. But maybe it also participates in geo-blocking of Russian channels in general, I don't know.

I can't even read their website myself, it's also blocked lol.

![Screenshot_2024-12-10-17-39-46-56_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/user-attachments/assets/1eff8eaa-8697-46b7-900b-a0159f417c95)

**Comment by Dum4G** [comment, marked as answer]
> I have tried out quite a few Russian regional network links that used to be region free last year. Now they are region locked, through a company or service called Wink (my actual region is USA) Which keeps insisting the stream or site is only for Russia even with Russian VPN (I have tried Proton and Vyper for this purpose ... works fine on Russian state tv channels like Russia One) and even used a GPS spoofer. I have had mixed results with GPS spoofing so it could be that. Also, TV-IN.RU main stream is fine except during KHL matches ... (this is the regional channel for HC Metallurg, in Magnitogorsk) Any ideas?

Wink was never supposed to play outside of Russia and Belarus. Maybe your DNS is leaking your location. And even if you got proper VPN Wink requires a specific user agent to avoid banner
